cdc/owner.go

```diff
@@ -717,14 +717,31 @@ func (o *Owner) flushChangeFeedInfos(ctx context.Context) error {
 		}
 		return nil
 	}
+	staleChangeFeedID := make([]model.ChangeFeedID, 0)
+	gcSafePoint := uint64(math.MaxUint64)
+
+	// init the lower bound of gcSafePoint, the logical part is zero.
+	minGcSafePoint := oracle.EncodeTSO(oracle.GetPhysical(time.Now()) - (o.gcTTL * 1000))
+	// Try to get physical and logical timestamp from pd
+	pyTs, lgTs, err := o.pdClient.GetTS(ctx)
```

Comment:
Since it is safe to use a smaller TSO, can we add a TSO cache such as https://github.com/pingcap/ticdc/blob/109257825f7805bfc0f87b68d9ef7e969aa19161/cdc/kv/store_op.go#L83

Otherwise the `GetTS` could lead to heavy requests to PD, because `flushChangeFeedInfos` could be called frequently.

Reply:
Very reasonable suggestion. I've added a cache.